src/assets/server/rest/v1/schemas/tag/tag-assign.json

```diff
@@ -0,0 +1,29 @@
+{
+  "title": "Tag assignment",
+  "description": "",
+  "type": "object",
+  "properties": {
+    "visualID": {
+      "$ref": "tag.json#/definitions/visualID"
+    },
+    "description": {
+      "$ref": "tag.json#/definitions/description"
+    },
+    "active": {
```

Comment:
Should not be part of the Assign request

Reply:
Done